changelogs/unreleased/gh-9842-rpc-for-roles.md

```diff
@@ -1,3 +1,5 @@
 ## feature/config
 
 * Introduced `instance_uri()` method for `config` module (gh-9842).
+* Introduced module `experimental.connpool` and function
+  `connect_using_config()` (gh-9842).
```

Comment:
The `config:` prefix in commit messages regarding the connection pool module seems irrelevant. The same for the `## feature/config` changelog category.

Reply:
Changed to `experimental`.